src/ui/components/Button/styles.scss

```diff
@@ -0,0 +1,180 @@
+@import "~photon-colors/colors";
+
+@import "~amo/css/inc/vars";
+@import "~core/css/inc/mixins";
+@import "~ui/css/vars";
+
+$micro-font-size: 11px;
+$default-font-size: 13px;
+$puffy-font-size: 15px;
+
+/* Button */
+@mixin button(
+  $background: $blue-60,
+  $background-active: $blue-80,
+  $background-hover: $blue-70,
+  $border-color: transparent,
+  $border-radius: $ui-border-radius-l,
+  $color: $white
+) {
+  @include font-regular();
+
+  align-items: center;
+  background: $background;
+  border: 1px solid $border-color;
+  border-radius: 2px;
+  cursor: pointer;
+  display: inline-flex;
+  height: 32px;
+  justify-content: center;
+  line-height: 1.3;
+  margin: 0;
+  padding: 0 8px;
+  text-decoration: none;
+  transition-duration: all $transition-long, padding $transition-short;
+  transition-property: background-color border-color color;
+  transition-timing-function: $easing-curve;
+
+  .Icon {
+    @include margin-end(6px);
+
+    height: $default-font-size;
+    width: $default-font-size;
+  }
+
+  // We frustratingly target :link directly so we need this override.
```

Comment:
I thought one of the goals for refactoring was to try using cascades instead of mixins so that these overrides wouldn't be needed. Did cascades not work out?

Reply:
Oh wait, this is because we target link styles using `:link` which messes with usual cascades and such... it's gonna have to stay for now (this was here before, just moved), but yeah until we can nix https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/Addon/styles.scss it's gotta stay.

I'd like to do that but it should be it's own issue as there's a large possibility of regressing link styles.